apps/src/netsim/netsim.js

```diff
@@ -964,11 +964,16 @@ function resizeFooterToFitToLeftOfContent() {
     return;
   }
 
+  var padding = parseInt(window.getComputedStyle(smallFooter)["padding-left"]);
+
+  var boundingWidth;
   if (leftColumn && $(leftColumn).is(':visible')) {
-    smallFooter.style.maxWidth = leftColumn.getBoundingClientRect().right + 'px';
+    boundingWidth = leftColumn.getBoundingClientRect().right;
   } else if (instructions && $(instructions).is(':visible')) {
-    smallFooter.style.maxWidth = instructions.getBoundingClientRect().right + 'px';
+    boundingWidth = instructions.getBoundingClientRect().right;
   }
+
+  smallFooter.style.maxWidth = (boundingWidth) ? (boundingWidth - padding) + 'px' : null;
```

Comment:
Ahh yes. At one point I had removed the padding, and then later added it back but didn't account for that here. Thanks for fixing.

In an ideal world, I think instead of manipulating the DOM for the small footer component externally we would actually be calling React.render again and passing in the new style (i.e. a style that includes maxWidth). We are of course not currently in said ideal world, so this will have to do. lgtm